scrapers/scrape_be_districts.py

```diff
@@ -1,79 +1,39 @@
+    246: 'Bern-Mittelland',
+    247: 'Thun',
+    248: 'Obersimmental-Saanen',
+    249: 'Frutigen-Niedersimmental',
+    250: 'Interlaken-Oberhasli',
 }
 
-# https://www.jgk.be.ch/jgk/de/index/gemeinden/gemeinden/gemeindedaten.assetref/dam/documents/JGK/AGR/de/Gemeinden/Gemeindedaten/agr_gemeinden_gemeindedaten_karte_verwaltungskreise_verwaltungsregionen_de.pdf
-inhabitants = {
-    'Jura bernois': 53721,
-    'Biel/Bienne': 101313,
-    'Seeland': 74467,
-    'Oberaargau': 81759,
-    'Emmental': 97218,
-    'Bern-Mittelland': 414658,
-    'Thun': 107491,
-    'Obersimmental-Saanen': 16588,
-    'Frutigen-Niedersimmental': 40375,
-    'Interlaken-Oberhasli': 47387,
-}
-
-
-# start getting and parsing the data
-html_url = 'https://www.besondere-lage.sites.be.ch/de/start/news/fallzahlen.html'
-d = sc.download(html_url, silent=True)
-d = d.replace('&nbsp;', ' ')
-soup = BeautifulSoup(d, 'html.parser')
-caption = soup.find('caption', string=re.compile('Fallzahlen nach Verwaltungskreisen'))
-table = caption.find_parent('table')
-trs = table.find_all('tr')
-weeks = []
-year = None
-years = []
-for th in trs[0].find_all('th'):
-    week = sc.find(r'Kalenderwoche (\d+)', th.text)
-    if week:
-        weeks.append(week)
-    year = sc.find(r'\d+\.\d+\.(\d+)', th.text)
-    if year:
-        if len(year) == 2:
-            year = f'20{year}'
-        years.append(year)
-
-assert len(weeks) == len(years), f'expected weeks ({weeks}) to have the same size as the years ({years})'
-
-tbody = table.find_next('tbody')
-for tr in trs[1:]:
-    tds = tr.find_all('td')
-    assert len(tds) == len(weeks), f'expected {len(weeks) + 1} items, but got {len(tds)}'
-    ths = tr.find_all('th')
-    district = ths[0].text
-    if district == 'Unbekannt':
-        continue
-    assert district in district_ids, f'district "{district}" not found in {district_ids}!'
-
-    for i in range(len(week)):
-        dd = sc.DistrictData(district=district, canton='BE')
-        dd.url = html_url
-        dd.district_id = district_ids[district]
-        dd.population = inhabitants[district]
-        dd.week = weeks[i]
-        dd.year = years[i]
-        dd.new_cases = tds[i].text.replace("'", "")
-        print(dd)
+url = 'https://covid-kennzahlen.apps.be.ch/#/de/cockpit'
+csv_url = 'https://raw.githubusercontent.com/openDataBE/covid19Data/develop/7_d_inzidenz_verwaltungskreis.csv'
+d = sc.download(csv_url, silent=True)
+reader = csv.DictReader(StringIO(d), delimiter=',')
+for row in reader:
+    #dd = sc.DistrictData(district=district, canton='BE')
+    district_id = int(row['bfs_nummer'])
+    dd = sc.DistrictData(district=district_ids[district_id], canton='BE')
+    dd.url = url
+    dd.district_id = district_id
+    dd.population = row['einwohnerzahl']
+    date = sc.date_from_text(row['datum'])
+    week = date.isocalendar()[1]
+    dd.week = week
+    dd.year = date.year
+    dd.new_cases = round(float(row['7_d_inzidenz']) / 100e3 * float(row['einwohnerzahl']))
```

Comment:
```suggestion
    dd.new_cases = round(float(row['7_d_inzidenz']) / 100e3 * int(row['einwohnerzahl']))
```

The result of this should be an integer, right? Or is there something I'm missing?

Reply:
Yep, it's an integer - you are of course right! I've applied your suggestion, thank you.